### ISSUE TITLE
Fix Travis

### ISSUE THREAD

**Issue by IDragonfire** [FAForever/client#190]
Travis is not building

```
py.test --cov src --cov-report term-missing
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/_pytest/config.py", line 543, in importconftest
    mod = conftestpath.pyimport()
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/py/_path/local.py", line 641, in pyimport
    __import__(modname)
  File "/home/travis/build/FAForever/lobby/conftest.py", line 5, in <module>
    sip.setapi('QString', 2)
ValueError: API 'QString' has already been set to version 1
ERROR: could not load /home/travis/build/FAForever/lobby/conftest.py
The command "py.test --cov src --cov-report term-missing" exited with 4.

```
from sheeo:

It's an environment change on travis
Build broke between these: https://travis-ci.org/FAForever/client/builds/45165618 and https://travis-ci.org/FAForever/client/builds/53633488

Only diff is this: https://github.com/FAForever/client/compare/a833777bbb48...1b923cc0ba89

###
sry ... hit enter to fast xD

**Comment by IDragonfire** [comment]
the travis changelog said nothing:
http://changelog.travis-ci.com/

**Comment by vinzBad** [comment]
maybe it's caused by an update in the python-dependencies

**Comment by vinzBad** [comment]
I set all packages in the requirements.txt to specific versions, which fixes this error 
https://travis-ci.org/vinzBad/client/builds/56403979
https://github.com/vinzBad/client/commit/42ba5f74b48e26a4270ebbf00771dc6ba30e0f96

Now the build failes with: "ConftestImportFailure: (local('/home/travis/build/vinzBad/client/tests/fa/conftest.py'), (<type 'exceptions.ImportError'>, ImportError('No module named _pygit2',), <traceback object at 0x2b39830>))"

**Comment by IDragonfire** [comment]
That is maybe easier to fix ...
We use our own pygit in https://github.com/FAForever/client/tree/develop/lib
Looks like Travis is not able to find this ...

**Comment by Sheeo** [comment]
Nice work @vinzBad.

We're thinking about moving away from pygit2, since it's so unstable.

**Comment by vinzBad** [comment]
Thanks @Sheeo this is my first contribution to an OSS - project :D

**Comment by JeroenDeDauw** [comment]
@vinzBad and hopefully not the last :)

**Comment by vinzBad** [comment]
@JeroenDeDauw currently trying vagrant to run those tests locally :) Is there a place where I can find more information about your build processes and how you create releases? I already looked at readme.md and the files for travis-ci. 
Do you discuss everything dev related via github or are there other places to ask questions?

**Comment by IDragonfire** [comment]
We have atm no Vagrant for the client, only for the server (since some days),
because currently we focus more to fix and improve the server side ...

**Comment by vinzBad** [comment]
I fixed some errors and created some new by deleting the local pygit2-build and installing it from source.
https://travis-ci.org/vinzBad/client/builds/56497774
https://github.com/vinzBad/client/commit/69bf0e6cb98dc4eb5e08fabb117722aaefac8d9b
Should I downgrade pygit2 or rewrite the failling tests? (I'm in favor of upgrading the tests)

@IDragonfire When I sign up for slack do have to select FAForever as company name or how does this work?

**Comment by Sheeo** [comment]
@vinzBad if you send your email to @visionik he will send you an invite to slack

Work on getting pygit2 to a usable state would be much apreciated!

**Comment by vinzBad** [comment]
Builds are back to green! 
https://travis-ci.org/vinzBad/client/builds/56522568
https://github.com/vinzBad/client/commit/61407d4c7b23a723993052074c954bf514045a36

What should I do before creating a pullrequest?

**Comment by Sheeo** [comment]
Nice job!

Looking at your commit history, you pushed all the fixes to the same branch (develop).

I would prefer if you put 36a1577 into it's own pull request for review.

**Comment by IDragonfire** [comment]
@vinzBad An interesting post for understanding git branching:
http://nvie.com/posts/a-successful-git-branching-model/

If you have one branch for each PR. you can push (or force push) to the branch to modify the PR ;)

A good example is this PR:
https://github.com/FAForever/server/pull/55
I fixed stuff after discussion

But don't look at my commit messages xD
https://github.com/FAForever/fa#contributing

**Comment by vinzBad** [comment]
This issue was resolved with #194

**Comment by vinzBad** [comment]
* #195
